### ISSUE TITLE
`status` command shows too much with `-vF`

### ISSUE THREAD

**Issue by ingydotnet** [ingydotnet/git-subrepo#84]
The `-v` option for `status` shows ref information. The `-v` for other commands
shows internal steps. This is all correct.

The `-F` flag for `status` fetches ref info, but when used with `-v` errantly
shows the steps too.

**Comment by stsquared99** [comment]
Should this be done, considering our decision in regards to --all vs --ALL? The code for --verbose is very clean right now, we would have to add an exception. The ref information isn't very much, perhaps that should just be printed by default?

**Comment by ingydotnet** [comment]
I'm not sure. I played around with it and I like the -v status vs normal and -q.

Let's wait on this for now.

**Comment by stsquared99** [comment]
I agree, I do like that functionality as well, it just seems awkward in function and in code to change the -v behavior.

In the meantime, can I send you a PR for the show vs status #163, or should I wait until the rest of the current status issues are resolved?

**Comment by ingydotnet** [comment]
Please go ahead with #163. Thanks!
